blaze/object_hook.py

```diff
@@ -9,7 +9,6 @@
 
 from datashape import dshape
 import numpy as np
-# these are used throughout blaze, don't remove them
 import pandas as pd
 
```

Comment:
Unused imports below `builtins, reduce`

Question: So `object_hook` no longer supports map/apply - Is that right? Looks like the json for following objects correctly converts them back to python objects but these wouldn't be used by map/apply

```
In [2]: object_hook._converters
Out[2]: 
{'__!bytes': <function blaze.object_hook._read_bytes>,
 '__!datashape': <function datashape.util.dshape>,
 '__!datetime': pandas.tslib.Timestamp,
 '__!frozenset': frozenset,
 '__!mono': <function blaze.object_hook._read_mono>,
 '__!timedelta': <function blaze.object_hook._read_timedelta>}
```